### DISCUSSION TITLE
2021-04-29 Dev Platform

### DISCUSSION THREAD

**Discussion by mikedotexe** [near/community#56] [Archived]
Updates related to Dev Platform team: https://github.com/near/dev-platform
Project board: https://github.com/orgs/near/projects/10
Tooling board: https://github.com/orgs/near/projects/13

## Done
- Chad having extended time off.
- `create-near-app` CI now using GitHub Actions.
- NEAR CLI tickets completed from Serhii as shown in the Tooling board linked above.
- Merged part of the `near-api-js` error handling.
- Token farm launched! https://tkn.farm

## In Progress
- Documentation, with Austin.
- Josh and Ryan continuing on the tooling tickets linked above.
- Upgrading rust toolchain, upgrading macro with new compilation errors.

**Comment by mfiyalka** [comment]
@mikedotexe How to add generated tokens to your wallet? Previously, the [tkn.farm](https://tkn.farm) website had an "Add to wallet" button

**Reply by mikedotexe** [reply]
Hello, my friend, I am no longer at NEAR, but I think you might find answers at https://near.chat
